editor/src/components/canvas/controls/select-mode/select-mode-hooks.tsx

```diff
@@ -595,7 +596,12 @@ function useSelectOrLiveModeSelectAndHover(
         }
       }
 
-      if (isDragIntention || hasInteractionSessionWithMouseMoved || !active || !isLeftClick) {
+      if (
+        isDragIntention ||
+        hasInteractionSessionWithMouseMoved ||
+        !active ||
+        !(isLeftClick || isRightClick)
+      ) {
```

Comment:
We should move this to a function with the sub-expressions broken up into separate variables, because this is already a little hard to comprehend for me